Parsedown.php

```diff
@@ -1217,7 +1217,7 @@ protected function inlineLink($Excerpt)
             return;
         }
 
-        if (preg_match('/^[(]((?:[^ ()]++|[(][^ )]+[)])++)(?:[ ]+("[^"]*"|\'[^\']*\'))?[)]/', $remainder, $matches))
+        if (preg_match('/^[(]\s*+((?:[^ ()]++|[(][^ )]+[)])++)(?:[ ]+("[^"]*"|\'[^\']*\'))?\s*[)]/', $remainder, $matches))
```

Comment:
Possessive quantifier – probably not necessary here actually, I tend to use them by habit if I know backtracking won't be needed